src/swap/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Swapping specific events.
+ */
+export enum SwapEventName {
+  SWAP_AUTOROUTER_VISUALIZATION_EXPANDED = 'Swap Autorouter Visualization Expanded',
+  SWAP_DETAILS_EXPANDED = 'Swap Details Expanded',
+  SWAP_MAX_TOKEN_AMOUNT_SELECTED = 'Swap Max Token Amount Selected',
+  SWAP_MODIFIED_IN_WALLET = 'Swap Modified in Wallet',
+  SWAP_PRICE_UPDATE_ACKNOWLEDGED = 'Swap Price Update Acknowledged',
+  SWAP_QUOTE_RECEIVED = 'Swap Quote Received',
+  SWAP_SIGNED = 'Swap Signed',
+  SWAP_SUBMITTED_BUTTON_CLICKED = 'Swap Submit Button Clicked',
+  SWAP_TOKENS_REVERSED = 'Swap Tokens Reversed',
+  SWAP_TRANSACTION_COMPLETED = 'Swap Transaction Completed',
+}
+
+export enum SwapPriceUpdateUserResponse {
```

Comment:
Wondering why we did not split these into different files either

Reply:
agreed i prefer diff files here